### PR TITLE
create view evaluation administration screen

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create View Evaluation Administration Screen #40](https://github.com/nerubia/kh360-react/issues/40)

### CODE REVIEW

File: src/features/admin/evaluation-administrations/EvaluationAdministrationsTable.tsx [hunk 1]
```diff
@@ -27,6 +28,10 @@ export const EvaluationAdministrationsTable = () => {
     )
   }, [searchParams])
 
+  const handleViewEvaluation = (id: number | undefined) => {
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/EvaluationAdministrationsTable.tsx:31]: remove undefined

File: src/features/admin/evaluation-administrations/[id]/preview/PreviewEmployeesTable.tsx [hunk 2]
```diff
@@ -74,7 +75,7 @@ export const PreviewEmployeesTable = () => {
             )}
           </table>
         </div>
-        <div className='border-t border-gray-300 w-full'></div>
+        <Divider orientation='horizontal' />
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/preview/PreviewEmployeesTable.tsx:78]: remove orientation, default is horizontal

**Review by Dev-Nino — APPROVED** [review]

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

**Review by jatnerubia — APPROVED** [review]
Nice

**Pull request merged**